src/main/java/shop/service/impl/UserServiceImpl.java

```diff
@@ -23,6 +30,9 @@ public UserResponseDto register(UserRegistrationRequestDto registerUserRequestDt
             throw new RegistrationException("Can't register user with given email");
         }
         User user = userMapper.toUser(registerUserRequestDto);
+        user.setPassword(passwordEncoder.encode(registerUserRequestDto.getPassword()));
+        user.setRoles(Set.of(roleRepository.findByRole(Role.RoleName.ROLE_USER)
+                .orElseThrow(() -> new EntityNotFoundException("Can`t find role"))));
```

Comment:
Let's make the code more explicit and the exception messages more informative
```suggestion
RoleName userRoleName = Role.RoleName.ROLE_USER
        user.setRoles(Set.of(roleRepository.findByRoleName(userRoleName)
                .orElseThrow(() -> new EntityNotFoundException("Can`t find role: " + userRoleName.name()))));
```